jsapp/js/components/common/tooltip.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export interface TooltipProps {
+  text: string;
+  ariaLabel: string;
+  alignment?: 'right' | 'left' | '';
```

Comment:
Let's have `export type TooltipAlignment = 'right' | 'left' | 'center';`. This way we can reference it elsewhere (e.g. in `button.tsx`) without the need to duplicate ourselves.

Reply:
Also `'center'` is better than empty string with comments "it means centered" :)